src/core/redux/selectors/selectUserSearchResults.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * @flow
+ */
+
+import { Map, getIn } from 'immutable';
+
+import { USERS, USER_SEARCH_RESULTS } from '../constants';
+
+export default function selectUserSearchResults() {
+
+  return (state :Map) :Map<string, Map> => getIn(state, [USERS, USER_SEARCH_RESULTS]) || Map();
+}
```

Comment:
annotation looks a little fuzzy

Reply:
What do you mean? The return value is a `Map` where the key is `string` and the value is `Map`.

Reply:
ah ok! makes sense!